src/conf_mode/protocols_isis.py

```diff
@@ -103,6 +112,10 @@ def verify(isis):
                               f'Recommended area lsp-mtu {recom_area_mtu} or less ' \
                               '(calculated on MTU size).')
 
+        if 'tun' in interface:
+            if not isis.get('tunnel_config').get(interface).get('remote'):
```

Comment:
Please stick to the common pattern and use `vyos.utils.dict_search()`

Reply:
done